### ISSUE TITLE
Stopped process after 0 or 1 iteration : 1it [00:10, 10.08s/it]stopped process

### ISSUE THREAD

**Issue by NathanDunand** [nerdyrodent/VQGAN-CLIP#123]
When I launch this cmd : 
``python3` generate.py -p "A painting of an apple in a fruit bowl"`` 
The program print :
``
Warning: No GPU found! Using the CPU instead. The iterations will be slow.
Perhaps CUDA/ROCm or the right pytorch version is not properly installed?
Working with z of shape (1, 256, 16, 16) = 65536 dimensions.
loaded pretrained LPIPS loss from taming/modules/autoencoder/lpips/vgg.pth
VQLPIPSWithDiscriminator running with hinge loss.
Restored from checkpoints/vqgan_imagenet_f16_16384.ckpt
Using device: cpu
Optimising using: Adam
Using text prompts: ['A painting of an apple in a fruit bowl']
Using seed: 103380167375222975
``

And after 10 seconds ~ the process stop and print : 
``
i: 0, loss: 0.933387, losses: 0.933387
1it [00:09,  9.37s/it]Processus arrêté
``
NB : "stopped process" in english I guess.

My conda version is 4.11.0 and others versions are :
- cpuonly
- pytorch==1.9.0
- torchaudio==0.9.0
- torchvision==0.10.0

Interesting fact : when I lauch a less powerful require cmd like ``python3 generate.py -p "A painting of an apple in a fruit bowl" -s 150 150 -i 10`` output.png is generated.
So I think that my cpu is not powerful enough but error message is not very verbose.

I'm on Ubuntu 20.04 LTS and my cpu is : 
``
Architecture: x86_64
Processor operating mode(s): 32-bit, 64-bit
Boutism: Little Endian
Address sizes: 39 bits physical, 48 bits virtual
Processor(s): 8
Processor(s) list online: 0-7
Thread(s) per core: 2
Core(s) per socket: 4
Socket(s): 1
NUMA Node(s): 1
Manufacturer ID: GenuineIntel
Processor family: 6
Model: 142
Model Name: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
Revision: 10
Processor Speed in MHz: 1800.000
Maximum Processor Speed in MHz: 3400.0000
Minimum processor speed in MHz: 400.0000
BogoMIPS: 3600.00
Virtualization: VT-x
L1d cache: 128 KiB
L1i cache: 128 KiB
L2 cache: 1 MiB
L3 cache: 6 MiB
NUMA 0 node of processor(s): 0-7
``

**Comment by tavisc** [comment]
So you don't have a GPU, and are running  integrated graphics? 

I think the unfortunate answer is your CPU just isn't powerful to create anything sizable, which is why restricting your image size works. You could try bumping up the size using progressively larger -i measurements but I would be surprised if you could get anything much larger than say, 200x200.

**Comment by NathanDunand** [comment]
After researches and tests I found that my CPU is powerful enough instead of my RAM. I have 8 Go RAM by default but I can add swap memory. With a 15Go swap file I can process 1080*1080px image, my CPU will work for hours (19 hours estimated) but it's works. It's not viable for me but it's not the CPU it's RAM.
Thus, my theory seems valid because when I visualize my RAM occupation (only 8Go RAM) during the execution, the percentage grow until 100% and the program crash with the same error.
